mix.exs

```diff
@@ -99,22 +99,23 @@ defmodule DotCom.Mixfile do
       {:inflex, "~> 1.8.0"},
       {:jason, "~> 1.1"},
       {:laboratory, [github: "paulswartz/laboratory", ref: "cookie_opts"]},
-      {:logster, "~> 0.4.0"},
-      {:mail, "~> 0.2"},
+      {:logster, "~> 1.0"},
```

Comment:
We should make these explicit rather than implicit with the patch version number missing. `1.0.0` instead of `1.0`. Are we sure we can't upgrade to at least `1.1.1`?